client/app/states/templates/editor/editor.state.js

```diff
@@ -0,0 +1,40 @@
+/* eslint camelcase: "off" */
+
+/** @ngInject */
+export function TemplatesEditorState(routerHelper) {
+  routerHelper.configureStates(getStates());
+}
+
+function getStates() {
+  return {
+    'templates.editor': {
+      url: '/edit/:templateId',
+      templateUrl: 'app/states/templates/editor/editor.html',
```

Comment:
Tried it and it doesn't seem to work the same way as it does for components.  Do you have an example of a state we are doing this in?